### DISCUSSION TITLE
GitHub Student Developer Error

### DISCUSSION THREAD

**Discussion by ghost** [community/community#45632] [GitHub Education]
Hello,

I am getting an error to complete my GitHub profile, which is already complete, but I am still getting the error as I submit the application. I have tried numerous times, but to no avail. So it would be great if anyone could assist me.

Regards
![image](https://user-images.githubusercontent.com/62020209/215272643-828efa3b-fdc0-4cde-857e-53219a3f8ea0.png)

**Comment by Neilblaze** [comment]
Did you apply with your `educational email` i.e. (say) `xyz@edu.in`. It does take some time to rollout the approval, so don't be tensed, and try again later!

**Reply by ghost** [reply]
My school doesn't have an education email, so I instead submitted my school ID, but I get the same error.

**Comment by Arturbl** [comment]
1) Make sure all the fields in your profile are filled out correctly and that all required fields have been completed.
2) Check if your email address is verified on GitHub and your app email is the same.
3) Verify that you don't have any outstanding issues or pull requests on your profile that need to be addressed.

**Reply by ghost** [reply]
1. All fields are filled up and completed.
2. Yes, the email is verified, and the app is using the same email.
3. I verified that I don't have any outstanding issues or pull requests on my profile.
